wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/service/EreignisartModel.java

```diff
@@ -0,0 +1,22 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis.EreignisartEntity;
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis.EreignisartDTO;
+
+public enum EreignisartModel {
+    VORFALL, VORKOMMNIS;
+
+    public static EreignisartModel fromEntity(EreignisartEntity entity) {
+        return switch (entity) {
+        case VORFALL -> VORFALL;
+        case VORKOMMNIS -> VORKOMMNIS;
+        };
+    }
+
+    public static EreignisartModel fromDto(EreignisartDTO entity) {
+        return switch (entity) {
+        case VORFALL -> VORFALL;
+        case VORKOMMNIS -> VORKOMMNIS;
+        };
+    }
```

Comment:
Bitte in den jeweiligen Mapper oder Testdatenfactory verlagern

Reply:
siehe commit [b1a086e](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/b1a086e3245ecd720cc3083f9e237c025716cf4d)